web_src/js/features/repo-settings.js

```diff
@@ -13,17 +13,17 @@ export function initRepoSettingsCollaboration() {
     const $text = $dropdown.find('> .text');
     $dropdown.dropdown({
       async action(_text, value) {
-        const lastValue = $dropdown.attr('data-last-value');
+        const lastValue = e.getAttribute('data-last-value');
```

Comment:
These are all wrong. You meant `e.target` instead of `e`. I doubt this was properly tested.

Reply:
`e` is an HTMLElement, not an event

Reply:
Ah ok. can you please rename to `el`?

Reply:
Done